src/mm/test.rs

```diff
@@ -9,16 +9,23 @@
 #[cfg(test)]
 mod tests {
 	use super::*;
+	use alloc::alloc::alloc;
 	use core::alloc::Layout;
 	use std::mem::{align_of, size_of};
 	use std::prelude::v1::*;
 
 	use crate::mm::allocator::*;
 	use crate::mm::hole::*;
 
+	#[cfg(target_arch = "x86_64")]
+	const CACHE_LINE_SIZE: usize = 128;
+	#[cfg(not(target_arch = "x86_64"))]
+	const CACHE_LINE_SIZE: usize = 64;
+
```

Comment:
Is it possible to reuse the existing definitions?

Reply:
I don't known an existing definition.

Reply:
I was referring to the ones you defined before

Reply:
Ah, yes, that is true. I will fix it